app/components-react/obs/ObsForm.tsx

```diff
@@ -216,21 +316,272 @@ export function ObsFormGroup(p: IObsFormGroupProps) {
+
+    filtered.splice(2, 0, 'Audio');
+    return filtered;
+  }, [p.sections]);
+
+  const [currentTab, setCurrentTab] = useState(p.sections[1].nameSubCategory);
+
+  return (
+    <div className="section" key="tabbed-section" style={{ marginBottom: '24px' }}>
+      {p.sections.map((sectionProps, ind) => (
+        <div className="section-content" key={`${sectionProps.nameSubCategory}${ind}`}>
+          {sectionProps.nameSubCategory === 'Untitled' && (
+            <>
+              <ObsForm
+                value={sectionProps.parameters}
+                onChange={formData => p.onChange(formData, ind)}
+              />
+              <Tabs tabs={tabs} onChange={setCurrentTab} style={{ marginBottom: '24px' }} />
+            </>
           )}
-          <div className="section-content">
+
+          {sectionProps.nameSubCategory === currentTab && (
             <ObsForm
               value={sectionProps.parameters}
-              onChange={formData => onChangeHandler(formData, ind)}
+              onChange={formData => p.onChange(formData, ind)}
             />
-          </div>
+          )}
+
+          {currentTab === 'Audio' && sectionProps.nameSubCategory.startsWith('Audio - Track') && (
+            <div
+              style={{
+                backgroundColor: 'var(--section-wrapper)',
+                padding: '15px',
+                marginBottom: '30px',
+                borderRadius: '5px',
+              }}
+            >
+              <h2 className="section-title">{$t(sectionProps.nameSubCategory)}</h2>
+              <ObsForm
+                value={sectionProps.parameters}
+                onChange={formData => p.onChange(formData, ind)}
+              />
+            </div>
+          )}
         </div>
       ))}
     </div>
   );
 }
+
+interface IObsCollapsibleFormGroupProps {
+  section: ISettingsSubCategory;
+  onChange: (formData: TObsFormData, ind: number) => unknown;
+}
+
+const { Panel } = Collapse;
+
+export function ObsCollapsibleFormItem(p: IObsCollapsibleFormGroupProps) {
+  const [expanded, setExpanded] = useState(true);
+
+  return (
+    <Collapse
+      className={cx('section-content', 'section-content--collapse')}
+      onChange={() => setExpanded(!expanded)}
+      expandIcon={({ isActive }) => (
+        <i
+          className={cx('fa', 'section-title-icon', {
+            'fa-minus': isActive,
+            'fa-plus': !isActive,
+          })}
+        />
+      )}
+      bordered={false}
+    >
+      <Panel
+        className="section-content--panel"
+        header={p.section.nameSubCategory}
+        key={`${p.section.nameSubCategory}`}
+      >
+        <ObsForm value={p.section.parameters} onChange={p.onChange} />
+      </Panel>
+    </Collapse>
+  );
+}
+
+interface IObsInputListCustomInputProps {
+  inputProps: any;
+  options?: Omit<IObsListOption<unknown>, 'description'>[];
+}
+
+const ObsInputListCustomInput = forwardRef<{}, IObsInputListCustomInputProps>((p, ref) => {
```

Comment:
hmm im not sure this component should be named so generically when it has a lot of code dedicated to handling custom resolutions specifically

Reply:
Renamed